src/Foundatio/Utility/OptionsBuilder.cs

```diff
@@ -1,17 +1,28 @@
 ﻿using System;
 
 namespace Foundatio {
-    public interface IOptionsBuilder<out T> {
-        T Target { get; }
+    public interface IOptionsBuilder {
+        object Target { get; }
+    }
+
+    public interface IOptionsBuilder<T> : IOptionsBuilder {
```

Comment:
The generic argument should be with `out` modifier to be covariant.

Reply:
It seems like I had a problem with it being out, but I will try again.